### PR TITLE
docs: add major release line prep doc

### PR DESCRIPTION
**Pull request by codebytere**
Per meeting discussion Wednesday, i've added a preliminary version of a Major Release Prep doc based on some of the tasks outlined [here](https://github.com/electron/electron/projects/20).

Please feel free to comment and leave feedback with any thoughts or things you feel may be missing here!

cc @electron/wg-releases

### CODE REVIEW

**Comment by codebytere** [comment]
@malept done!

File: wg-releases/major-release-process.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+After a new major line has been officially stabilized, we should begin to prepare for the beta process of the next line.
+
+For this, we need to ensure the following:
+
+* We have a new Node Module Version (NMV) for the line that follows what is about to become the new `master` major
+  * If we've just released `v6.0.0`, this means that `7-0-x` is about to become the beta line, and so we would need to procure a NMV for `8-0-x`
+* Sudowoodo is capable of handling betas for the new major line
+* [`node-abi`](https://github.com/lgeiger/node-abi) beta update
+  * We should ensure that  abi version information is added for the first beta in the new major stabilization line
```
Comment (malept) [inline wg-releases/major-release-process.md]:
```suggestion
  * We should ensure that ABI version information is added for the first beta in the new major stabilization line
```

File: wg-releases/major-release-process.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* The blog post has been successfully published to the [Electron Website](https://electronjs.org)
+
+## Subsequent Release Line Preparations
+
+After a new major line has been officially stabilized, we should begin to prepare for the beta process of the next line.
+
+For this, we need to ensure the following:
+
+* We have a new Node Module Version (NMV) for the line that follows what is about to become the new `master` major
+  * If we've just released `v6.0.0`, this means that `7-0-x` is about to become the beta line, and so we would need to procure a NMV for `8-0-x`
```
Comment (malept) [inline wg-releases/major-release-process.md]: Add a link or instructions for to how to procure a new NMV from the Node.js folks?

Comment (codebytere) [reply wg-releases/major-release-process.md]: It's a very ad-hoc process right now, unfortunately, which is why i didn't add super granular instructions here

Comment (malept) [reply wg-releases/major-release-process.md]: Oh, I take it the Node TSC hasn't codified a way to update [the ABI version registry](https://github.com/nodejs/node/blob/v12.6.0/doc/abi_version_registry.json) yet.

File: wg-releases/major-release-process.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Major Release Process
+
+This document outlines the major release stable promotion process.
```
Comment (MarshallOfSound) [inline wg-releases/major-release-process.md]: This through my head into a spin for a while 😄 

```suggestion
This document outlines the beta -> stable promotion process for a major release.
```

File: wg-releases/major-release-process.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Subsequent Release Line Preparations
+
+After a new major line has been officially stabilized, we should begin to prepare for the beta process of the next line.
+
+For this, we need to ensure the following:
+
+* We have a new Node Module Version (NMV) for the line that follows what is about to become the new `master` major
+  * If we've just released `v6.0.0`, this means that `7-0-x` is about to become the beta line, and so we would need to procure a NMV for `8-0-x`
+* Sudowoodo is capable of handling betas for the new major line
```
Comment (MarshallOfSound) [inline wg-releases/major-release-process.md]: Should we link to the bit that needs updating.  i.e. https://github.com/electron/sudowoodo/blob/master/src/slack-manager.ts#L240-L265

File: wg-releases/major-release-process.md (hunk 2 above)
Comment (MarshallOfSound) [inline wg-releases/major-release-process.md]: Currently this isn't true, our `6-0-x` branch and `master` branch (6 and 7) respectively have the same NMV number.  I've gone ahead and requested a new number for Electron 7 here --> https://github.com/nodejs/node/pull/28774

Once we release Electron 6 stable we should follow this document and request one for Electron 8

**Pull request merged**